### PR TITLE
BaSP-MR-110: Members redux development

### PR DESCRIPTION
**Pull request by FontanR**
Implementation of redux in members resource

Page should look the same way (no changes applied to the view) but now with redux we can handle a bit better the toast functionality.
Toast should disappear from the view a few seconds after it appears but this could be fast foward if the user clicked a link or over the toast itself.
In addition, a loader was added for the resource index while waiting for the api data to load that sohuld be displayed in the center of the page and with the same colors of the header and footer design.

### CODE REVIEW

**Review by francoax — CHANGES_REQUESTED** [review]
The button `Add new member` its shown when the table is loading and when you want to create or edit a member, the confirm modal doesn't disappear after click on the submit button if you have some mistakes in the form.

If you want, you can change the text for the button "Add new member" to  " + Add new " to continue with the consistency with the team

**Comment by francoax** [comment]
> The button `Add new member` its shown when the table is loading and when you want to create or edit a member, the confirm modal doesn't disappear after click on the submit button if you have some mistakes in the form.
> 
> If you want, you can change the text for the button "Add new member" to " + Add new " to continue with the consistency with the team

However, everything works perfect !! Good job

File: src/Components/Members/Form/index.jsx [hunk 1]
```diff
@@ -100,10 +51,7 @@ const MemberForm = ({ match }) => {
           password: data.password
         });
       } catch (error) {
-        handleToast(true, 'fail', error.message);
-        setTimeout(() => {
-          handleShowToast(false);
-        }, 1500);
+        console.log(error);
```
Comment (MauroCaffesse) [inline src/Components/Members/Form/index.jsx]: Please delete this console.log()

**Review by MauroCaffesse — CHANGES_REQUESTED** [review]

**Review by MauroCaffesse — APPROVED** [review]
Great Job!!

**Review by francoax — APPROVED** [review]

**Pull request merged**